packages.yml

```diff
@@ -1,4 +1,7 @@
 packages:
-  - package: fivetran/iterable_source
-    version: [">=0.8.0", "<0.9.0"]
-
+  # - package: fivetran/iterable_source
+  #   version: [">=0.8.0", "<0.9.0"]
+  # - local: ../dbt_iterable_source
+  - git: https://github.com/fivetran/dbt_iterable_source.git
```

Comment:
Reminder to switch the package version back before release.

Reply:
Reminder to switch before merge and release